bazel/repository_locations.bzl

```diff
@@ -705,8 +705,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Common Expression Language (CEL) C++ library",
         project_desc = "Common Expression Language (CEL) C++ library",
         project_url = "https://opensource.google/projects/cel",
-        version = "9841e3ee251f3cc4cd5b6dd9deee6818bc9f2854",
-        sha256 = "7e42cbad7d1068d6e7891ad101e2863e727692136d6b3a817c487b3cc7bcfdcc",
+        version = "3b7f7061d80cff882dc477faf74a2cd525ee9b62",
+        sha256 = "c4e38e791a6a56042cd15e06e025ab70e8ae0dcd2d7c3c3278c606f79b33f0eb",
```

Comment:
As per the PR title can we switch to releases?
```
https://github.com/google/cel-cpp/archive/refs/tags/v0.5.0.tar.gz
293b4a0479a4390e08b24e7f14a307e4dd94495367b104c4a37926a760597aa7
2021-03-31
```

Reply:
Done.